operatorhub/openshift/config.yaml

```diff
@@ -209,7 +209,7 @@ image-substitutions:
         containerName: openshift-pipelines-operator-lifecycle
         envKeys:
           - IMAGE_CHAINS_TEKTON_CHAINS_CONTROLLER
-- image: registry.redhat.io/                                                                                                          @sha256:1dbb6d28b453f8f1b5bc52f4b3281c3d5c40500ad0a2f12a5b154e1774166b55
+- image: registry.redhat.io/rhel9/postgresql-13@sha256:ed119cb7ce29e287bde84d01ad33ebec783153c5a69d0357928df8ca10a22633
```

Comment:
The image was previously referencing postgresql-16, is the downgrade to 13 here intentional?

Reply:
> The image was previously referencing postgresql-16, is the downgrade to 13 here intentional?

I think there were some issues during the operator upgrade related to PostgreSQL 16. @ab-ghosh , can you please confirm?